spec/models/ext_management_system_spec.rb

```diff
@@ -18,12 +18,13 @@
 
   let(:all_types_and_descriptions) do
     {
-      "ansible_tower_configuration" => "Ansible Tower Configuration",
+      "ansible_tower_automation"    => "Ansible Tower Automation",
       "azure"                       => "Azure",
       "azure_network"               => "Azure Network",
       "ec2"                         => "Amazon EC2",
       "ec2_network"                 => "Amazon EC2 Network",
       "ec2_ebs_storage"             => "Amazon EBS",
+      "embedded_ansible_automation" => "Embedded Ansible Automation",
```

Comment:
I really don't like the name having Automation at the end.  @blomquisg Thoughts?  I understand it's because of the multiple manager thing, but it feels gross.  Even so, it should not hold up this PR.